### ISSUE TITLE
translate to indonesia for documentation vlang

### ISSUE THREAD

**Issue by slowy07** [vlang/v#14062]
greetings, i have some idea for vlang documentation, i wanna create pull request for creating documentation for installing and and tutorial in indonesia, hope some people from indonesia more know about how to install or tutorial about basic programming language, thankss! 👋🏻

**Comment by vincenzopalazzo** [comment]
there is a better option, we are working on a form of doc with gitbook, and it supports multilanguage, maybe it can be this https://github.com/vincenzopalazzo/vbook can better place where work on this feature?

**Comment by slowy07** [comment]
yes nice one 💯 , so for work i create unofficial books of vlang documentation for indonesia right ?, like https://github.com/vincenzopalazzo/vbook or create pull request for a while ?

thanks

**Comment by trufae** [comment]
How multilanguage is handled?

**Comment by blackshirt** [comment]
> greetings, i have some idea for vlang documentation, i wanna create pull request for creating documentation for installing and and tutorial in indonesia, hope some people from indonesia more know about how to install or tutorial about basic v programming language, thankss! 👋🏻
> 
> with filename `docs_id.md` for indonesia version

Its glad to know vlang going more in broader way,     and welcome @slowy07 ....

**Comment by slowy07** [comment]
it pleasure sir @blackshirt 👍🏻
